turbo/trie/trie_root.go

```diff
@@ -889,6 +902,18 @@ func (c *AccTrieCursor) _unmarshal(k, v []byte) {
 	if c.lvl >= len(k) {
 		from, to = len(k)+1, c.lvl+2
 	}
+
+	// Consider a trie DB with keys like: [0xa, 0xbb], then unmarshaling 0xbb
```

Comment:
Hopefully this comment explains it pretty well, but, when trie node `0xa` is processed, it is assigned to `c.k[0]`, then, when the next trie node `0xbb` is processed, because its length `2` is longer than `0xa`'s length `1`, the `from` is set to `2`, and the entry for `0xa` in `c.k[1]` is not cleared.  Thus, when looking for the sibling of a parent, the code incorrectly traverses back to this already processed node.